packages/internal/src/storage-test-utils.js

```diff
@@ -133,7 +139,10 @@ export const makeMockChainStorageRoot = () => {
   return Far('mockChainStorage', {
     ...bindAllMethods(rootNode),
     /**
-     * Defaults to deserializing pass-by-presence objects into Remotable objects.
+     * Defaults to deserializing slot references into plain Remotable
+     * objects, but if supplied with a different marshaller, it could
+     * produce Remotables with e.g. the slog string embedded in the
+     * iface.
```

Comment:
```suggestion
     * Defaults to deserializing slot references into plain Remotable
     * objects having the specified interface name (as from `Far(iface)`),
     * but can accept a different marshaller for producing Remotables
     * that e.g. embed the slot string in their iface name.
```

Reply:
works for me, but the PR landed already.. @turadg did you have more followup to do? wanna include this in yours?

Reply:
oh nevermnd, I see #7610